crates/trigger-http/src/lib.rs

```diff
@@ -612,7 +624,7 @@ mod tests {
             .build_trigger()
             .await;
 
-        let body = Body::from("Fermyon".as_bytes().to_vec());
+        let body = spin_http::full(Bytes::copy_from_slice("Fermyon".as_bytes()));
```

Comment:
Why don't we use [`Bytes::from_static`](https://docs.rs/bytes/1.5.0/bytes/struct.Bytes.html#method.from_static) for these?

Reply:
Because I didn't know (or forgot) about it.  I'll change them.